sunpy/cm/color_tables.py

```diff
@@ -341,14 +348,30 @@ def lasco_color_table(number):
         }[number]
     except KeyError:
         raise ValueError(
-            "Invalid LASCO number. Valid values are "
-            "2, 3."
+            "Invalid LASCO number. Valid values are 2, 3."
         )
 
     # Now create the color dictionary in the correct format
     cdict = create_cdict(r, g, b)
     return colors.LinearSegmentedColormap('SOHO LASCO C{:s}'.format(str(number)), cdict)
 
+
+def lasco_color_table(number):
+    """Returns one of the fundamental color tables for SOHO LASCO images.
+    These are not the same as those used in SSWIDL.  This is because the
+    SSWIDL color scaling for LASCO level 0.5 and 1.0 is highly compressed
+    and does not display the data well."""
+    if number == 2:
+        rgb = cm.get_cmap('gist_heat')(np.linspace(0, 1, 256)) * 255
```

Comment:
This seems... redundant?

The operation we want to do here is to give the same colourmap a different name. We can do that with a simple call to https://matplotlib.org/api/cm_api.html#matplotlib.cm.register_cmap

I think therefore my advice is to move the contents of this function directly into `cm/cm.py` and just call

```python
mpl.cm.register_cmap("SOHO LASCO C2", mpl.cm.get_cmap("gist_heat"))
```

Reply:
Also as an aside, we should probably not use `import matplotlib.cm as cm` in a sunpy submodule named `cm` where there is also a file called `cm` it get's very confusing very quick!!